src/azure-cli/azure/cli/command_modules/acr/_format.py

```diff
@@ -243,6 +292,33 @@ def _agentpool_format_group(item):
     ])
 
 
+def _connected_registry_format_group(item):
+    parent_id = _get_value(item, 'parent', 'id')
+    parent_name = '' if parent_id.isspace() else parent_id.split('/connectedRegistries/')[1]
+    return OrderedDict([
+        ('NAME', _get_value(item, 'name')),
+        ('MODE', _get_value(item, 'mode')),
+        ('CONNECTION STATE', _get_value(item, 'connectionState')),
+        ('PARENT', parent_name),
+        ('LOGIN SERVER', _get_value(item, 'loginServer', 'host')),
+        ('LAST ACTIVITY (UTC)', _get_value(item, 'lastActivityTime')),
+        ('LAST SYNC (UTC)', _get_value(item, 'parent', 'syncProperties', 'lastSyncTime')),
+        ('SYNC SCHEDULE', _get_value(item, 'parent', 'syncProperties', 'schedule')),
+        ('SYNC WINDOW', _get_value(item, 'parent', 'syncProperties', 'syncWindow'))
+    ])
+
+
+def _connected_registry_list_format_group(item):
+    return OrderedDict([
+        ('NAME', _get_value(item, 'name')),
+        ('MODE', _get_value(item, 'mode')),
+        ('CONNECTION STATE', _get_value(item, 'connectionState')),
+        ('PARENT', _get_value(item, 'parent_name')),
+        ('LOGIN SERVER', _get_value(item, 'loginServer_host')),
```

Comment:
Just curious as to why in the line 303 it _get_value(item, 'loginServer', 'host') and why the format is different in this list _get_value(item, 'loginServer_host')

Reply:
same question, properties in list of connected registry is different with above connected registry